src/charm.py

```diff
@@ -353,6 +355,10 @@ def _update(self, event):
                     }
                 },
             )
+
+            # Open port for cache warm-up.
+            self.model.unit.open_port(port=APPLICATION_PORT, protocol="tcp")
```

Comment:
Should this service port only be opened for a specific `charm-function`?

Reply:
Yes, only for the `app-gunicorn`